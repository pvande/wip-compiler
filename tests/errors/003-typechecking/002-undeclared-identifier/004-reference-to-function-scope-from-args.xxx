x := (y := a) => {
  a := "inner scope"
}
