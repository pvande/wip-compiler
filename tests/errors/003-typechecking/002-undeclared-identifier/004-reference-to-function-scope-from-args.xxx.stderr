

Error: "Could not infer the type of this variable"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/004-reference-to-function-scope-from-args.xxx[0m on line [1;37m1[0m

> [0;36mx := (y := a) => {[0m
  [0;31m      ^           [0m
